tests/integration/components/bs-modal-test.js

```diff
@@ -49,18 +49,18 @@ module('Integration | Component | bs-modal', function (hooks) {
 
   test('clicking ok button closes modal when autoClose=true with custom component hierarchy', async function (assert) {
     this.owner.register(
-      'component:my-component',
+      'component:test-component',
       class extends Component {
         layout = hbs`{{yield}}`;
       }
     );
 
     await render(hbs`
       <BsModal @title="Simple Dialog" @body={{false}} @footer={{false}} as |modal|>
-        {{#my-component}}
+        {{#test-component}}
```

Comment:
Seems as we have missed that one when refactoring to angle bracket component invocation. As we are touching it now, let's fix it:

```suggestion
        <TestComponent>
```